07_Big-Data/README.md

```diff
@@ -1,87 +1,196 @@
-# 7_ Big Data
+## 1_ MapReduce fundamentals
 
-Big Data refers to extremely large data sets that may be analyzed computationally to reveal patterns, trends, and associations. It involves the storage, processing, and analysis of data that is too complex or large for traditional data processing tools.
+MapReduce is a distributed programming model and processing framework used for processing and generating large datasets in parallel on clusters. It consists of two main phases: the Map phase, which applies a user-defined function to each input record and produces a set of intermediate key-value pairs, and the Reduce phase, which aggregates and combines the intermediate values associated with the same intermediate key.
```

Comment:
I quickly read it, this kind of PR looks really to be chatGPT generated. The definition are reworded, but not that much new information is brought. Also please do not change the table of content/titles as they refer to the map, which is the entire point of this repo.